components/Sidebar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+import useUser from 'hooks/useUser';
+import { clearToken } from 'helpers/authentication';
+
+export const sidebarTestIds = {
+  base: 'sidebar',
+  userName: 'sidebar__user-name',
+  button: 'sidebar__logout-button',
+};
+
+const Sidebar = ({ name }) => {
+  const { mutateUser } = useUser({
+    redirectTo: '/login',
+  });
+
+  const handleOnClick = async event => {
+    event.preventDefault();
+    clearToken();
+    mutateUser();
+  };
+
+  return (
+    <div
+      className="fixed bg-sidebar backdrop-blur-4xl right-0 w-[264px] pl-8 pt-8 pr-8 h-full"
+      data-test-id={sidebarTestIds.base}
+    >
+      <div className="text-white text-base-xxl font-extrabold mb-[22px]" data-test-id={sidebarTestIds.userName}>
```

Comment:
Should we adjust the bottom margin to `20px` ?

```suggestion
      <div className="text-white text-base-xxl font-extrabold mb-[20px]" data-test-id={sidebarTestIds.userName}>
```

Current style | The design
--- | ---
![Screen Shot 2565-04-11 at 15 21 21](https://user-images.githubusercontent.com/1772999/162695137-58946670-c219-490f-8766-3dc06af56a97.png) | ![Screen Shot 2565-04-11 at 15 18 53](https://user-images.githubusercontent.com/1772999/162695167-23fd4264-f202-46f2-b637-f24ce9deea54.png)

Reply:
done in 2125544666199978509632db13f61e3cd5903fa6